src/status_im/ui/screens/browser/views.cljs

```diff
@@ -74,31 +90,38 @@
   (let [domain-name (nth (re-find #"^\w+://(www\.)?([^/:]+)" url) 2)]
     (get (:inject-js browser-config) domain-name)))
 
+(defn navigation [webview browser can-go-back? can-go-forward?]
+  [react/view styles/toolbar
+   [react/touchable-highlight {:on-press            #(re-frame/dispatch [:browser-nav-back browser])
+                               :disabled            (not can-go-back?)
+                               :style               (when-not can-go-back? styles/disabled-button)
+                               :accessibility-label :previou-page-button}
```

Comment:
Typo: `previous`

Reply:
thanks

Reply:
btw this is not my change, we have this long time @antdanchenko ?

Reply:
well, we are using the same typo https://github.com/status-im/status-react/blob/746d580e3406cceb0707ddb30652203789801607/test/appium/views/web_views/base_web_view.py#L31 

please, change it in both places if you are going to change it in this PR
otherwise I can change it in one of my PRs